dynatrace-operator/templates/Common/Routing/serviceaccount-routing.yaml

```diff
@@ -0,0 +1,8 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes or openshift" (include "dynatrace-operator.platformSet" .))}}
```

Comment:
The roles, etc. are missing this platform check.

Reply:
Added